procstats/delaystats_linux.go

```diff
@@ -0,0 +1,32 @@
+package procstats
+
+import (
+	"errors"
+	"syscall"
+
+	"github.com/segmentio/taskstats"
+)
+
+func collectDelayInfo(pid int) (info DelayInfo, err error) {
+	client, err := taskstats.New()
+	if err != nil {
+		if err1, ok := err.(syscall.Errno); ok && err1 == syscall.ENOENT {
```

Comment:
This type check is unnecessary, you could just check for `if err == syscall.ENOENT`